erpnext/controllers/selling_controller.py

```diff
@@ -67,38 +67,6 @@ def set_price_list_and_item_details(self, for_validate=False):
 		self.set_price_list_currency("Selling")
 		self.set_missing_item_details(for_validate=for_validate)
 
-	def apply_shipping_rule(self):
```

Comment:
where is this method added back???

Reply:
Looks like last time I rebased something went wrong, it is in accounts_controller.py file.

Reply:
Thats why we need test cases! can you add a few test cases to ensure the Shipping rule works for both Sales and Purchase ?

Reply:
@rmehta Test cases done for both selling and buying but travis is failing again for some other reasons. :(

Reply:
👍 will check other issues